From 8b21a83bd8728e3c89ae2c8688befc64b2ae0ee4 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Mon, 6 Mar 2023 12:58:16 +0100
Subject: libstdc++: std::vector cannot contain const members in
 password_importer.cc

libstdc++ implementation of std::vector forbids explicitly
declaration with const members.

Bug: 957519
Change-Id: I7cd9926dee4b5979ebc97326d9e12daa152a3293
---
 .../password_manager/core/browser/import/password_importer.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/components/password_manager/core/browser/import/password_importer.cc b/components/password_manager/core/browser/import/password_importer.cc
index 0a500dc516905..3e98daff18196 100644
--- a/components/password_manager/core/browser/import/password_importer.cc
+++ b/components/password_manager/core/browser/import/password_importer.cc
@@ -218,7 +218,7 @@ void AddCredentialsCallback(
 }
 
 bool HasConflicts(
-    const std::map<std::u16string, std::vector<const CredentialUIEntry>>&
+    const std::map<std::u16string, std::vector<CredentialUIEntry>>&
         credentials_by_username,
     const CredentialUIEntry& imported_credential) {
   auto it = credentials_by_username.find(imported_credential.username);
@@ -339,7 +339,7 @@ void PasswordImporter::ConsumePasswords(
   add_credentials.reserve(seq->csv_passwords.size());
 
   // Used to compute conflicts and duplicates.
-  std::map<std::u16string, std::vector<const CredentialUIEntry>>
+  std::map<std::u16string, std::vector<CredentialUIEntry>>
       credentials_by_username;
   for (const CredentialUIEntry& credential : presenter_->GetSavedPasswords()) {
     // Don't consider credentials from a store other than the target store.
-- 
2.37.2

